d_max: 6.4
d_min: 1.0
dt: 0.01
grid_res: 0.1
h_max_above_ground: 1.0
hm_interp_method: null
integration_mode: rk4
k_damping: 894.4271909999159
k_friction: 0.5
k_stiffness: 5000.0
n_sim_trajs: 32
omega_max: 0.4
robot_I:
- - 30.0
  - -8.940696716308594e-07
  - 0.0
- - -8.940696716308594e-07
  - 47.85713577270508
  - 0.0
- - 0.0
  - 0.0
  - 67.85713958740234
robot_mask_left:
- true
- true
- true
- true
- true
- true
- true
- true
- false
- false
- false
- false
- false
- false
- false
- false
robot_mask_right:
- false
- false
- false
- false
- false
- false
- false
- false
- true
- true
- true
- true
- true
- true
- true
- true
robot_mass: 40.0
robot_points:
- - -0.5
  - 0.25
  - 0.20000000298023224
- - -0.3571428656578064
  - 0.25
  - 0.10000000149011612
- - -0.2142857015132904
  - 0.25
  - 0.0
- - -0.07142855226993561
  - 0.25
  - 0.0
- - 0.07142855226993561
  - 0.25
  - 0.0
- - 0.2142857015132904
  - 0.25
  - 0.0
- - 0.3571428656578064
  - 0.25
  - 0.10000000149011612
- - 0.5
  - 0.25
  - 0.20000000298023224
- - -0.5
  - -0.25
  - 0.20000000298023224
- - -0.3571428656578064
  - -0.25
  - 0.10000000149011612
- - -0.2142857015132904
  - -0.25
  - 0.0
- - -0.07142855226993561
  - -0.25
  - 0.0
- - 0.07142855226993561
  - -0.25
  - 0.0
- - 0.2142857015132904
  - -0.25
  - 0.0
- - 0.3571428656578064
  - -0.25
  - 0.10000000149011612
- - 0.5
  - -0.25
  - 0.20000000298023224
robot_size:
- 1.0
- 0.5
traj_sim_time: 5.0
vel_max: 1.2
